README.md

```diff
@@ -201,5 +201,5 @@ transactions:
 The ledger format schema is purposely lightweight. The only requirements are as follows:
 - the `account` field should be expressed in the following format: `account_classification:account_name`.
 - the `amount` field should be a number. It can include up to two (2) decimal points.  
-- the `date` field should be in the following format: `MM-DD-YYYY`. 
+- the `date` field should be in the following format: `YYYY-MM-DD`. 
```

Comment:
Will the previous format still work or will users be required to update their yaml files?

Reply:
Unfortunately, no. This format is relied upon in the current implementation. I have been experimenting locally with the `chrono` lib to handle date "types" so that the logic can be more generic, but I haven't found an easy way to make this happen without a larger implementation.

Admittedly, the date format change is primarily driven by personal preference. Understandably, the recent ledger file schema changes would make it frustrating for anyone using this downstream, but then again, this tool is still a long way from being useful for a larger audience, so I think it is okay to make this change in the meantime.

Reply:
as an update, I revisited the `chrono` work I had locally and I think I have it working. It still doesn't fix the generic date problem as YAML date strings that don't fit into the `YYYY-MM-DD` format make the `serde` deserialization logic panic, but I think this is a good change that will make working with dates (particularly sorting and filtering) much easier going forward.

Reply:
This looks like a really good change. I also prefer to have the date in this format.